apimodels/agent_models.go

```diff
@@ -200,6 +201,30 @@ type Token struct {
 	Token string `json:"token"`
 }
 
+type AssumeRoleRequest struct {
+	RoleARN         string `json:"role_arn"`
+	Policy          string `json:"policy"`
+	DurationSeconds *int32 `json:"duration_seconds"`
```

Comment:
Is there a particular reason this is an int pointer? Is 0 different from nil/the default?

Reply:
If it makes passing the value easier and 0 vs nil are treated the same, then I think it's fine.